From: =?utf-8?q?Timo_R=C3=B6hling?= <roehling@debian.org>
Date: Wed, 20 Apr 2022 20:03:10 +0200
Subject: Revert commit f745e049 temporarily

This commit introduced a regression with qa=+canary; it has
been reverted upstream pending further investigation.

Forwarded: https://gitlab.kitware.com/cmake/cmake/-/issues/23432
---
 Modules/CMakeCheckCompilerFlagCommonPatterns.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Modules/CMakeCheckCompilerFlagCommonPatterns.cmake b/Modules/CMakeCheckCompilerFlagCommonPatterns.cmake
index 8f59acd..bda1d71 100644
--- a/Modules/CMakeCheckCompilerFlagCommonPatterns.cmake
+++ b/Modules/CMakeCheckCompilerFlagCommonPatterns.cmake
@@ -13,7 +13,6 @@ macro (CHECK_COMPILER_FLAG_COMMON_PATTERNS _VAR)
     FAIL_REGEX "unknown .*option"                          # Clang
     FAIL_REGEX "optimization flag .* not supported"        # Clang
     FAIL_REGEX "unknown argument ignored"                  # Clang (cl)
-    FAIL_REGEX "warning: .* ignored"                       # Clang (linker)
     FAIL_REGEX "ignoring unknown option"                   # MSVC, Intel
     FAIL_REGEX "warning D9002"                             # MSVC, any lang
     FAIL_REGEX "option.*not supported"                     # Intel
